.github/ISSUE_TEMPLATE/add_partner_form_template.md

```diff
@@ -0,0 +1,29 @@
+---
+name: 🏢 Add Partner Form
+about: Create a request to add a partner in the website
+title: "[PARTNER]"
```

Comment:
Just want to start a thread about alternative names for partner, since that might imply a higher bar than what we have now. @avocadosand @stockholmux 

Everyone seems to use the term partner though.

Reply:
Some possible alternatives:
- Collaborating companies
- Commercial support
- Commercial offerings

Reply:
A recommendation came for "Participants", which I like a bit more than partner.

Reply:
I will update it later today

Reply:
done